src/locales/en.json

```diff
@@ -717,6 +717,7 @@
   "mno_enterprise.templates.dashboard.provisioning.confirm.name": "Name",
   "mno_enterprise.templates.dashboard.provisioning.confirm.description": "Description",
   "mno_enterprise.templates.dashboard.provisioning.confirm.edit_order": "Edit Order",
+  "mno_enterprise.templates.dashboard.provisioning.confirm.update_cart": "Update Cart",
```

Comment:
@MAhsenArif Was this added to mno-enterprise?

Reply:
I think it got missed. Adding it quickly